### DISCUSSION TITLE
Problem with running movie quick add script

### DISCUSSION THREAD

**Discussion by SigmaQuadrat** [chhoumann/quickadd#553] [Q&A] [answered]
![image](https://github.com/chhoumann/quickadd/assets/73792015/7c197bea-1f41-486a-917a-f6350b675c2f)

When I run the quick add script in Obsidian i get an error: Failed to run user script and then no input given.
I then need to fill out all the IMDB info manually via pop up windows.
I tried using it on an offline vault and one on google cloud but with the same resault.

I would really appreciate it  if someone could help me with this problem.

**Comment by chhoumann** [comment, marked as answer]
401 means there's an issue with your API key.

**Reply by SigmaQuadrat** [reply]
Thank you I figured it out
